schema/instance.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:generate stringer -type=Type -linecomment
```

Comment:
Since we use `-linecomment` we  need to add `// avro` to the end of the constant in line 22.